test/functional/cmdLineTests/J9security/src/com/ibm/j9/jsr292/indyn/bootpath/CodeNotTrusted.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package com.ibm.j9.jsr292.indyn.bootpath;
+
+import static java.lang.invoke.MethodType.methodType;
+
+import java.lang.invoke.CallSite;
+import java.lang.invoke.ConstantCallSite;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodType;
+import java.lang.invoke.MethodHandles.Lookup;
+
+public class CodeNotTrusted {
+	public static CallSite bootstrap_test_privAction(Lookup lookup, String name, MethodType type) throws Throwable {
+		System.out.println("user.home = " + System.getProperty("user.home"));
+
+		MethodHandle handle = lookup.findStatic(CodeNotTrusted.class, "voidMethod", methodType(void.class));
```

Comment:
```suggestion
		MethodHandle handle = MethodHandles.constant(Object.class, null).asType(MethodType.methodType(void.class))
```

This avoids doing the `findStatic` which is more complex.  This should work all the time, with no security implications

Reply:
Updated.